### PR TITLE
fix(docker/digitalocean-pgbouncer): Switch from forceTransactions to pgbouncer

### PR DESCRIPTION
**Pull request by janpio**
This PR adapt the e2e tests to our new PgBouncer configuration.

### CODE REVIEW

**Comment by steebchen** [comment]
Removed myself, I don't have anything useful to add here. Please ask @divyenduz to review.

File: databases/digitalocean-pgbouncer/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# PGBouncer on DigitalOcean
+# PgBouncer on DigitalOcean
 
-We want to test DigitalOcean and pgBouncer setup with Prisma client. Prisma client constructor has a `forceTransactions` flag to enable support for pgBouncer.
+We want to test DigitalOcean and PgBouncer setup with Prisma Client. Prisma Query Engine has a `pgbouncer=true` query param to enable support for pgBouncer.
```
Comment (divyenduz) [inline databases/digitalocean-pgbouncer/README.md:3]: 1. We still need custom configuration. What prompted us to change from client constructor parameter (`forceTransactions`) to a query string parameter? 

2. Are there other parameters that are Prisma specific and not native to the database?

File: databases/docker-pgbouncer/prisma/schema.prisma [hunk 2]
```diff
@@ -1,6 +1,6 @@
 datasource db {
   provider = "postgresql"
-  url      = "postgresql://postgres:postgres@127.0.0.1:6433/blog?schema=public"
+  url      = "postgresql://this-will-not-be-used-but-overwritten-in-the-prisma-client-constructor"
```
Comment (divyenduz) [inline databases/docker-pgbouncer/prisma/schema.prisma:3]: Somewhat confusing that this is defined in the schema but not being used by Prisma client.

File: databases/digitalocean-pgbouncer/README.md (hunk 1 above)
Comment (janpio) [reply databases/digitalocean-pgbouncer/README.md:3]: > We still need custom configuration

What does this refer to?

> What prompted us to change from client constructor parameter (forceTransactions) to a query string parameter?

See internal discussion: https://prisma-company.slack.com/archives/C1FPU5FPT/p1589560375303200

> Are there other parameters that are Prisma specific and not native to the database?

Yes, the connection limits etc.

File: databases/docker-pgbouncer/prisma/schema.prisma (hunk 2 above)
Comment (janpio) [reply databases/docker-pgbouncer/prisma/schema.prisma:3]: Suggestion on how to improve this? This makes it more explicit that the two Clients have their own connection string in my opinion.

**Pull request merged**